records_mover/session.py

```diff
@@ -156,6 +158,34 @@ def _boto3_session(self) -> boto3.session.Session:
         else:
             return self.creds.boto3_session(self._default_aws_creds_name)
 
+    def set_stream_logging(self,
+                           app_name: str = 'records_mover',
+                           default_root_log_level: int = logging.INFO,
+                           stream: IO[str] = sys.stdout,
+                           fmt: str = '%(asctime)s - %(message)s',
+                           datefmt: str = '%H:%M:%S') -> None:
+        """
+        records-mover logs details about its operations using Python logging.  This method is a
+        simple way to configure that logging to be output to a stream (by default, stdout).
+
+        :param app_name: Name of the application using records-mover.  If set to 'foo', you can set
+        a log variable FOO_LOG_LEVEL to the log level threshold you'd like to set
+        (INFO/WARNING/etc).
+        :param default_log_level: Logging more detailed than this will not be output to the stream.
+        :param stream: Stream which logging should be sent (e.g., sys.stdout, sys.stdin, or perhaps
+        a file you open)
+        :param fmt: Logging format to send to Python'slogging.Formatter() - determines what details
+         will be sent.
+        :param datefmt: Date format to send to Python'slogging.Formatter() - determines how the
+        current date/time will be recorded in the log.
+        """
+        log_levels.set_levels(app_name, default_root_log_level)
+        formatter = logging.Formatter(fmt, datefmt)
+        handler = logging.StreamHandler(stream=stream)
+        handler.setFormatter(formatter)
+        root_logger = logging.getLogger()
```

Comment:
It looks like this is setting application-wide logging, rather than logging for just the records_mover package. In contrast, [the boto logging helpers](https://github.com/boto/boto3/blob/develop/boto3/__init__.py#L37) add a logging handler only to the named logger (which defaults to the boto3 logger). I would suggest we follow that pattern. If folks desire app-wide logging, they can just call logging.basicConfig-- seems like this helper should be targeted towards helping people log just records mover output easily. What do you think?

Reply:
Makes sense!